ArticleTemplates/assets/scss/layout/_article--shared.scss

```diff
@@ -52,6 +52,9 @@
         .element-interactive {
             width: auto;
 
+            // Needed to solve tablet bug in interactive-now-and-then-embed
```

Comment:
We could give more detail here. Something like "This prevents uneven sizes of now-and-then images in interactive embeds on Android tablets".